packages/jest-circus/src/types.ts

```diff
@@ -183,6 +183,7 @@ export type State = {
   currentlyRunningTest: TestEntry | undefined | null; // including when hooks are being executed
   expand?: boolean; // expand error messages
```

Comment:
not for this PR, but we should use tsdoc for these so they show up when using autocomplete